### PR TITLE
Fix for ubuntu 14.04

### PR DESCRIPTION
**Pull request by steliosmilidonis**
The installation fails on ubutnu 14.04 , by installing nodejs 6 the installation runs very smooth

### CODE REVIEW

**Comment by PalanteJon** [comment]
@totten While I could bikeshed this PR, it solves the problem and I'd recommend merging.  See also #315, which is this same problem on Debian Jessie.

**Comment by totten** [comment]
I'm OK with this but would like comment from @AkA84 -- I think you said that the Shoreditch build depended on a particular version of `node`? Do you know which it was?

**Comment by AkA84** [comment]
@totten yes it's v5.12.0 (see [here](https://github.com/civicrm/org.civicrm.shoreditch/blob/develop/CODING.md#build)). I haven't followed the issue, so i'm not sure if this is the most correct approach to the issue we can find. If it is, then I imagine that one could simply use [nvm](https://github.com/creationix/nvm) to *also* install v5.12.0 when he needs to work on Shoreditch

**Comment by totten** [comment]
@steliosmilidonis @PalanteJon would node v5 work?

**Comment by steliosmilidonis** [comment]
yes it would

File: bin/civi-download-tools [hunk 1]
```diff
@@ -321,14 +321,16 @@ function do_system_precise() {
...
-    PACKAGES="acl git wget unzip zip mysql-server mysql-client php5-cli php5-imap php5-ldap php5-curl php5-mysql php5-intl php5-gd php5-mcrypt php-apc apache2 libapache2-mod-php5 nodejs-legacy npm"
+    PACKAGES="acl curl git wget unzip zip mysql-server mysql-client php5-cli php5-imap php5-ldap php5-curl php5-mysql php5-intl php5-gd php5-mcrypt php-apc apache2 libapache2-mod-php5"
     echo "Detected \"Ubuntu Trusty 14.04\"."
     echo ""
     echo "Recommended packages: $PACKAGES"
     echo ""
     if cvutil_confirm "Run automated installation? [Y/n] " y y; then
       sudo apt-get update
       sudo apt-get -y install $PACKAGES
+      sudo curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
```
Comment (PalanteJon) [inline bin/civi-download-tools]: If you change `6.x` to `5.x` it sounds like this will get merged.

**Comment by Hassan3152** [comment]
it works for me after doing the below steps:
1-run this script curl -Ls https://civicrm.org/get-buildkit.sh > get-buildkit.sh
2-search for (function do_system_trusty) and find (PACKAGES=) then remove nodejs-legacy and npm from this line and save it.
3-rub the rest of the step one script: bash civicrm-buildkit.sh --full --dir ~/buildkit
4-install nodejs 5 using this command: 
curl -sL https://deb.nodesource.com/setup_5.x | sudo -E bash - 
and the last step to setup nodejs 5 : sudo apt-get install -y nodejs

**Pull request merged**